Assets/Scripts/StandardSamples/UI/Storage/Title/UITitleStorageMenu.cs

```diff
@@ -46,7 +46,7 @@ public class UITitleStorageMenu : SampleMenu
         public GameObject UIFileNameEntryPrefab;
 
         public Text FileContent;
-        private EOSTitleStorageManager TitleStorageManager;
+        //private TitleStorageService TitleStorageManager;
```

Comment:
Can this be removed now that it isn't needed?

Reply:
Yup, done.